src/cloudevents.rs

```diff
@@ -0,0 +1,811 @@
+// SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+// [impl->dsn~cloudevents-umessage-mapping~2]
+
+use crate::{
+    UAttributes, UAttributesError, UAttributesValidators, UCode, UMessage, UMessageError,
+    UMessageType, UPayloadFormat, UPriority, UUri, UUID,
+};
+use bytes::Bytes;
+use protobuf::{well_known_types::any::Any, Enum, EnumOrUnknown, MessageField};
+
+pub use cloudevents::{cloud_event::CloudEventAttributeValue, CloudEvent};
+
+include!(concat!(env!("OUT_DIR"), "/cloudevents/mod.rs"));
+
+pub const CONTENT_TYPE_CLOUDEVENTS_PROTOBUF: &str = "application/cloudevents+protobuf";
```

Comment:
This is failing our tests (because symbol is unused)

Reply:
already on it